src/index.js

```diff
@@ -1,4 +1,5 @@
-let debug = require('debug')('contexture:mongo')
+import debug from 'debug'
+debug('contexture:mongo')
```

Comment:
This won't work. You need to use whatever is returned by `debug`. Ex:

```javascript
import _debug from 'debug'
let debug = _debug('contexture:mongo')
```

Reply:
Oh yeah good catch, I'll udpate it.